go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	github.com/pulumi/pulumi-random/sdk/v4 v4.16.0
 	github.com/pulumi/pulumi-tls/sdk/v4 v4.11.1
 	github.com/pulumi/pulumi/sdk/v3 v3.108.1
+	github.com/pulumiverse/pulumi-time/sdk v0.0.0-20231010123146-089d7304da13
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You can remove the explicit `pulumi-time` plugin install call in https://github.com/DataDog/test-infra-definitions/blob/main/Dockerfile#L112